includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -580,6 +580,10 @@ private function maybe_add_lightbox_attributes( $attributes, $node ) {
 	private function is_gif_url( $url ) {
 		$ext  = self::$anim_extension;
 		$path = wp_parse_url( $url, PHP_URL_PATH );
+		// 检查$path是否为null，若是则赋值为空字符串
```

Comment:
For this project, code comments are in English. In any case, no need for a comment here since the reason is self-explanatory:

```suggestion
```